pkgs/top-level/all-packages.nix

```diff
@@ -23552,6 +23552,8 @@ with pkgs;
 
   spaceship-prompt = callPackage ../shells/zsh/spaceship-prompt {};
 
+  sparrow3d = callPackage ../development/libraries/sparrow3d {};
```

Comment:
Split sparrow3d into its own commit (no need to open another PR).

In case of doubts, use this tutorial:
https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

Reply:
okay, i've split the commit